Add query to SPARQLwrapper strategy

torhaugl opened the pull request:
# Description
Add query to SPARQLwrapper backend. The result follows the same format as rdflib backend.

Can be tested with the following Python code if you have a SPARQL endpoint,
```python
from tripper import SparqlwrapperStrategy
endpoint = "YOUR_ENDPOINT" #something like r"http://IP:PORT/repositories/TOPMAST"
ts = SparqlwrapperStrategy(endpoint)
res = ts.query(r"SELECT ?s ?p ?o WHERE { ?s ?p ?o . } LIMIT 1000")
print(res)
```

## Type of change
- [ ] Bug fix and code cleanup
- [x] New feature
- [ ] Documentation update
- [ ] Testing


## Checklist for the reviewer
This checklist should be used as a help for the reviewer.

- [ ] Is the change limited to one issue?
- [ ] Does this PR close the issue?
- [ ] Is the code easy to read and understand?
- [ ] Do all new feature have an accompanying new test?
- [ ] Has the documentation been updated as necessary?
- [ ] Is the code properly tested?

jesper-friis commented:
> Can be tested with the following Python code if you have a SPARQL endpoint,
> 
> ```python
> from tripper import SparqlwrapperStrategy
> endpoint = "YOUR_ENDPOINT" #something like r"http://IP:PORT/repositories/TOPMAST"
> ts = SparqlwrapperStrategy(endpoint)
> res = ts.query(r"SELECT ?s ?p ?o WHERE { ?s ?p ?o . } LIMIT 1000")
> print(res)
> ```

The example should be rewritten as

```python
from tripper import Triplestore
endpoint = "YOUR_ENDPOINT" #something like r"http://IP:PORT/repositories/TOPMAST"
ts = Triplestore(backend="sparqlwrapper", base_iri=endpoint)
res = ts.query(r"SELECT ?s ?p ?o WHERE { ?s ?p ?o . } LIMIT 20")
print(res)
```

Works great!

jesper-friis commented on tripper/backends/sparqlwrapper.py:
```diff
@@ -39,6 +39,31 @@ def __init__(self, base_iri: str, **kwargs) -> None:
         self.sparql = SPARQLWrapper(endpoint=base_iri, **kwargs)
 
+    def query(
+        self, query_object, **kwargs
```
Silence the pylint error
```suggestion
        self, query_object, **kwargs  # pylint: disable=unused-argument
```

jesper-friis replied:
It is a good idea to install pre-commit. In your development virtualenv do the following

    pip install -U -e .[dev]
    pre-commit install
    pre-commit run --all  # Force to run all checks

Pre-commit is run together with pytest as CI tests on GitHub, so to avoid failing CI tests it is a good idea to run both before pushing.

jesper-friis reviewed (commented):
Works nicely. But please have a look at the comments.

torhaugl commented:
I have added a test and adressed the comments. In addition I added a simple SPARQL query test which requires an internet connection. Please have a new look
